test/elixir_analyzer/exercise_test/assert_call/module_tracking_test.exs

```diff
@@ -0,0 +1,335 @@
+
+        def function() do
+          {:ok, file} = Utils.read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        import Elixir.{Mix, Hex.Utils}
+
+        def function() do
+          {:ok, file} = Utils.read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        import Elixir.Mix.Utils, only: :functions
+
+        def function() do
+          {:ok, file} = read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        import Elixir.Mix.Utils, only: [read_path: 1]
+
+        def function() do
+          {:ok, file} = read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        import Elixir.Mix.Utils, only: [print_tree: 3]
+
+        def function() do
+          {:ok, file} = Elixir.Mix.Utils.read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        import Elixir.Mix.Utils, except: [print_tree: 3]
+
+        def function() do
+          {:ok, file} = read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        import Elixir.Mix.Utils, except: [read_path: 0, read_path: 2, read_path: 3]
+
+        def function() do
+          {:ok, file} = read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        alias Elixir.Mix.Utils
+
+        def function() do
+          {:ok, file} = Utils.read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        alias Elixir.Mix
+
+        def function() do
+          {:ok, file} = Mix.Utils.read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        alias Elixir.{Hex.API, Mix, Mix.CLI}
+
+        def function() do
+          {:ok, file} = Mix.Utils.read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        alias Elixir.{Mix, Hex, Hex.Config}
+
+        def function() do
+          {:ok, file} = Mix.Utils.read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        alias Elixir
+
+        def function() do
+          {:ok, file} = Elixir.Mix.Utils.read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        alias Elixir.Mix.Utils, as: U
+
+        def function() do
+          {:ok, file} = U.read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        alias Elixir.Mix, as: M
+
+        def function() do
+          {:ok, file} = M.Utils.read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        alias Elixir, as: E
+
+        def function() do
+          {:ok, file} = E.Mix.Utils.read_path("mix.exs")
+        end
+      end,
+      defmodule AssertCallVerification do
+        alias Elixir.Mix.Utils, as: U
+
+        def function() do
+          {:ok, file} = Elixir.Mix.Utils.read_path("mix.exs")
```

Comment:
Was this meant to use the alias?
```suggestion
        alias Elixir.Mix.Utils, as: U

        def function() do
          {:ok, file} = U.Mix.Utils.read_path("mix.exs")
```

Reply:
No, here I meant to import an alias and check if the full path still works.